src/components/StatusBar.vue

```diff
@@ -67,6 +69,11 @@
                 Sel <span class="num">{{ selectionSize }}</span>
             </template>
         </div>
+        <div class="status-block line-number" v-if="updatedTime !== ''">
+            Updated <span class="num">{{ updatedTime }}</span>
+            <!-- Show the created time as well -- too verbose -->
+            <!-- <span v-if="createdTime !== updatedTime">&nbsp;&nbsp;(Created {{ createdTime }})</span> -->
```

Comment:
This looked too busy to me. Perhaps the better UI would be to click on the udpated time to see the created time? That wouldn't be clear either though.

Feel free to adjust this as you see it